sourcetools/com.ibm.jpp.preprocessor/com/ibm/jpp/commandline/CommandlineBuilder.java

```diff
@@ -209,6 +209,7 @@ private static void missingValueFor(String arg) {
 		validArgs.add("-verdict");
 		validArgs.add("-includeifunsure");
 		validArgs.add("-nowarn");
+		validArgs.add("-nowarnincludeif");
```

Comment:
Are there changes missing? Seems more than this is required to implement the new option.

Reply:
Keith pointed out the option is already implemented at https://github.com/eclipse-openj9/openj9/blob/master/sourcetools/com.ibm.jpp.preprocessor/com/ibm/jpp/om/Builder.java#L476

Reply:
The recognition of that option is here: https://github.com/eclipse-openj9/openj9/blob/master/sourcetools/com.ibm.jpp.preprocessor/com/ibm/jpp/om/Builder.java#L476.